tests/test_config.py

```diff
@@ -336,14 +328,20 @@ def test_env_var_save_and_load(cleanup_env_files):  # noqa: ARG001
     assert read_env_var("OTHER_VAR") == "other_value"
 
 
-def test_env_var_not_found(cleanup_env_files):  # noqa: ARG001
+def test_env_var_not_found(mock_env_file):  # noqa: ARG001
     """Test behavior when environment variable is not found"""
     assert read_env_var("NONEXISTENT_VAR") is None
```

Comment:
This tests fails if I do "export NONEXISTENT_VAR=test" before, but this is a rather artificial case...

Reply:
Yes this won't be the case at least in the github workers, and if it happens, we can adjust.